### PR TITLE
feat: allow specifying a path to flake.nix within the repository

### LINKED ISSUE #50: Allow specifying a path to flake.nix within the repository
I have a repository in which I don't have control over the layout of files. Unfortunately the `flake.nix` file is not located at the root of the repository. Would it be possible to add another input to this action such that I could specify a path within the repository where the `flake.nix` is located?

### PR DESCRIPTION
**Pull request by aanderse**
##### Description

Allow specifying a path to flake.nix within the repository - resolves #50. I ran a single test where I used the `path-to-flake-nix` input and it performed as expected.

##### Checklist

- [x] Tested functionality against a test repository (see ["How to test changes"](../README.md#how-to-test-changes))
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -17,6 +17,10 @@ inputs:
     description: 'The branch of the PR to be created'
     required: false
     default: "update_flake_lock_action"
+  path-to-flake-nix:
```
Comment (cole-h) [inline action.yml:20]: And also if you wouldn't mind naming this:

```suggestion
  path-to-flake-nix-dir:
```

It's a bit of a mouthful, to be sure, but I think it more accurately communicates what it should be set to, if people don't read the description (the input should be "the path to the directory containing the flake.nix", not "the path to the flake.nix itself").

**Review by cole-h — COMMENTED** [review]
Thanks for this! I wonder if I could cajole you into writing a brief section in the README, or even just add its usage to one of the existing examples.

File: action.yml (hunk 1 above)
Comment (cole-h) [reply action.yml:20]: Maybe it could even be:

```suggestion
  path-to-flake-dir:
```

**Comment by aanderse** [comment]
Thanks for the feedback. I'll address in the next day or two. 🚀

**Comment by aanderse** [comment]
@cole-h please feel free to suggest or make any additional changes.

**Review by cole-h — APPROVED** [review]
Oops, I was a bit busy yesterday and forgot to get back to you -- sorry!

LGTM!

**Comment by aanderse** [comment]
Thanks @cole-h!

**Pull request merged**